js/util/browser.js

```diff
@@ -1,29 +1,46 @@
 'use strict';
 
+/**
+ * Unlike js/util/browser.js, this code is written with the expectation
```

Comment:
This description was specific to the browser version of browser.js, and can be removed or rewritten.